helm/servicex/values.yaml

```diff
@@ -261,3 +261,7 @@ dataLifecycle:
   # See https://www.geeksforgeeks.org/date-command-linux-examples/#4-how-to-display-past-dates
   # for examples
   retention: "7 days ago"
+
+  # The cleanup service will go beyond the retention date to keep the server side cache size below
+  # this threshold. Specify this as a string with Mb, Gb, or Tb units in the string
+  maxDesiredCacheSize: "100Gb"
```

Comment:
This is rather small for a real deployment, no? Even our testing instances have O(400 GB) for the cache ...